certified-connectors/nunify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,466 @@
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Add Registrant",
+        "operationId": "ADD_REGISTRANT",
+        "description": "Registers an attendee to your event in nunify and issues a ticket for them.",
+        "parameters": [
+          {
+            "name": "Platform Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Platform Id",
+            "description": "This value can be copied from the API Keys page in your nunify event."
+          },
+          {
+            "name": "Domain Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Domain Id",
+            "description": "This value can be copied from the API Keys page in your nunify event."
+          },
+          {
+            "name": "App Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "App Id",
+            "description": "This value can be copied from the API Keys page in your nunify event."
+          },
+          {
+            "name": "map_by_labels",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "default": true,
+            "x-ms-visibility": "internal",
+            "x-ms-summary": "Map by Labels",
+            "description": "This is an internal parameter that is required to be set to true by the API."
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal",
+            "x-ms-summary": "Content Type",
+            "description": "Content Type header set to application/json to support JSON request/response exchange."
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal",
+            "x-ms-summary": "Accept",
+            "description": "Accept header set to application/json to support JSON request/response exchange."
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "email": {
+                  "type": "string",
+                  "description": "email",
+                  "title": "Email"
+                },
+                "first_name": {
+                  "type": "string",
+                  "description": "first_name",
```

Comment:
@nunify-admin All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.